geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractBitPosIntegrationTest.java

```diff
@@ -115,11 +123,69 @@ public void bitposWithStart_givenOneInLastByte() {
     assertThat(jedis.bitpos(key, true, new BitPosParams(-1))).isEqualTo(7 + 3 * 8);
   }
 
+  @Test
+  public void bitposWithStart_givenStartGreaterThanOrEqualToByteArrayLength() {
+    byte[] key = {1, 2, 3};
+    byte[] bytes = {1, 1, 1, 1};
+    jedis.set(key, bytes);
+    assertThat(jedis.bitpos(key, true, new BitPosParams(bytes.length))).isEqualTo(7 + 3 * 8);
+    assertThat(jedis.bitpos(key, true, new BitPosParams(bytes.length + 1))).isEqualTo(7 + 3 * 8);
+  }
```

Comment:
Looks like Native Redis doesn't check whether the start is greater than or equal to the length of the value beforehand and just returns -1 if start is greater than end:
```
    /* For empty ranges (start > end) we return -1 as an empty range does
     * not contain a 0 nor a 1. */
    if (start > end) {
        addReplyLongLong(c, -1);
```

I definitely feel like we've gotten beyond the scope of this Jira ticket/PR though.

Reply:
Agreed. I'll remove this test case. If/when bitpos becomes a supported command, proper coverage can be established.